src/be/app.py

```diff
@@ -24,20 +29,52 @@ def index():
 # serve the frontend's static assets
 @app.get("/static/{path:path}")
 def static(path: str):
+    print(f"static path: {path}")
     return fa.responses.FileResponse(
         path=_fe_dir / path,
     )
 
+# I don't know why this needs to be here but it breaks without it :)
+@app.get("/turn_on")
+def hello():
+    bulb.turn_on()
+    return "Hello from the server!"
 
 @app.get("/api/v1/status")
 def status():
+    #Use the `bulb` variable to get the properties of the LED strip, such as its name, power status, brightness, and color. Return these properties as an instance of the `Status` class.
+    bulb_name_variable = bulb.get_properties()['name']
+    power_status = bulb.get_properties()['power']
+    brightness = bulb.get_properties()['bright']
+    colour = bulb.get_properties()['rgb']
+    if bulb_name_variable is None:
```

Comment:
After taking into account the above comment, the default value of "Unknown" can be assigned with:

```python
bulb_name_variable = properties["name"] or "Unknown"
```
The `or` operator does more that just boolean logic, if interested [this](https://realpython.com/python-or-operator/#using-or-with-common-objects) seems like a decent write-up.